Source/WebCore/platform/graphics/gstreamer/eme/WebKitOpenCDMDecryptorGStreamer.cpp

```diff
@@ -202,6 +230,23 @@ static bool webKitMediaOpenCDMDecryptorDecrypt(WebKitMediaCommonEncryptionDecryp
         return false;
     }
 
+#ifdef USE_OPENCDM_BCM_NEXUS_SVP
+    B_Secbuf_Info secureBufferInfo;
+    void* opaqueData;
+
+    // If there is no subsample, only allocate one region for clear+enc, otherwise, number of subsamples.
+    uint32_t rpcSubSampleTotalSize = (subSampleCount ? subSampleCount * 2 : 2) * sizeof(uint32_t);
+    uint32_t sizeOfRPCInfo = sizeof(RPCSecureBufferInformation) + rpcSubSampleTotalSize;
+    RPCSecureBufferInformation* rpcSecureBufferInformation = reinterpret_cast<RPCSecureBufferInformation*> (g_alloca(sizeOfRPCInfo));
```

Comment:
`alloca` concerns me here since we have unbounded input. You normally see that used for things that are known to be less that a few hundred bytes. That isn't guaranteed here, although certainly it would be the common case. Better to be safe here and allocate on the heap.